src/components/views/loginPage/RegisterPage.vue

```diff
@@ -4,60 +4,67 @@
       <v-flex xs12 sm8 md8>
         <v-card class="elevation-12">
           <v-toolbar dark color="blue">
-            <v-toolbar-title>Signup form</v-toolbar-title>
+            <v-toolbar-title>{{ $language.register.signUpForm }}</v-toolbar-title>
           </v-toolbar>
           <v-card-text>
-            <v-form>
-              <v-alert
-                :value="userExists"
-                color="error"
-                icon="warning"
-              >This user already exists, try a different set of data.</v-alert>
+            <v-form form @submit.prevent="register()" id="signup-form">
+              <v-alert :value="!!alertMsg" color="error" icon="warning">{{alertMsg}}</v-alert>
 
               <v-text-field
-                prepend-icon="person"
-                name="login"
-                v-model="username"
-                label="Login"
+                name="privateName"
+                v-model="privateName"
+                :label="$language.register.privateName"
+                :rules="[rules.required]"
+              ></v-text-field>
+
+              <v-text-field
+                name="privateName"
+                v-model="lastName"
+                :label="$language.register.lastName"
                 :rules="[rules.required]"
               ></v-text-field>
 
               <v-text-field
-                prepend-icon="email"
                 name="email"
                 v-model="email"
-                label="Email"
+                :label="$language.register.email"
                 :rules="[rules.required, rules.email]"
               ></v-text-field>
 
               <v-text-field
-                prepend-icon="lock"
                 name="password"
-                label="Password"
+                :label="$language.register.password"
                 :rules="[rules.required]"
                 type="password"
                 v-model="password"
               ></v-text-field>
 
               <v-text-field
-                prepend-icon="lock"
                 name="password"
-                label="Confirm Password"
+                :label="$language.register.confirm_password"
                 :rules="[rules.required]"
                 type="password"
                 v-model="confirm_password"
                 :error="!valid()"
               ></v-text-field>
+
+              <v-divider light></v-divider>
+
+              <v-radio-group v-model="role">
+                <v-radio
+                  v-for="role in roles"
+                  :key="role.name"
+                  :label="`${role.name}`"
+                  :value="role.name"
+                ></v-radio>
+              </v-radio-group>
             </v-form>
           </v-card-text>
           <v-divider light></v-divider>
           <v-card-actions>
-            <v-btn to="/login" round color="black" dark>Login</v-btn>
+            <v-btn @click.prevent="openLigonModal()" color="black" dark>כבר נרשמת?</v-btn>
```

Comment:
Want to localize this as well?

Reply:
Will do. Thank you very much for the review